src/sass/modules/_m-button.scss

```diff
@@ -25,6 +25,12 @@ button,
   }
 }
 
+.usa-button-outline {
+  .usa-accordion &:focus {
+    box-shadow: inset 0 0 0 2px $color-primary-darkest, 0 0 3px $color-focus, 0 0 7px $color-focus;
+  }
+}
```

Comment:
With nesting something higher on the visual hierarchy with something lower, it for some reason made sense to me in my head to group it like that since that's the button module, but I don't feel strongly either way so I don't mind reversing it.

Reply:
Thanks for pointing this out @ncksllvn, since you mentioned that this has been resolved in newer versions of USWDS (which we are upgrading [to here](https://github.com/department-of-veterans-affairs/vets-website/pull/6299)), this way of structuring the nesting -- to accommodate additional overrides -- may not be necessary. Also, it may be helpful to include a comment to remove this once the upgrade has been performed.

Reply:
Sorry, saw this just now - but thanks @cehsu! I'll add a comment to that about the upgrade in just a sec!